Source/JavaScriptCore/API/JSValueRef.cpp

```diff
@@ -390,12 +390,11 @@ JSValueRef JSValueMakeFromJSONString(JSContextRef ctx, JSStringRef string)
     JSGlobalObject* globalObject = toJS(ctx);
     JSLockHolder locker(globalObject);
     String str = string->string();
-    unsigned length = str.length();
-    if (!length || str.is8Bit()) {
-        LiteralParser<LChar> parser(globalObject, str.characters8(), length, StrictJSON);
+    if (str.isEmpty() || str.is8Bit()) {
```

Comment:
Does `is8Bit` return true for empty strings?

Reply:
Indeed, looking at the code, I believe the isEmpty() check is unnecessary.